file-formats/dobj/dobj-v1.json

```diff
@@ -0,0 +1,134 @@
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "description": {
+      "title": "Description",
+      "description": "Description",
+      "type": "string",
+      "maxLength": 60
+    },
+    "dobjDetails": {
+      "title": "Data destruction object",
+      "description": "Data destruction object",
+      "type": "object",
+      "properties": {
+        "destrProgram": {
+          "title": "Destruction Program",
+          "description": "Destruction Program",
+          "type": "string",
+          "maxLength": 60
+        },
+        "applicComponent": {
+          "title": "Application Component",
+          "description": "Application Component",
+          "type": "string",
+          "maxLength": 30
+        },
+        "structureDefinition": {
+          "title": "Structure Definition",
+          "description": "Structure Definition",
+          "type": "array",
+          "uniqueItems": true,
+          "items": {
+            "title": "Structure Definition",
+            "description": "Structure Definition",
+            "type": "object",
+            "properties": {
+              "recordNo": {
+                "title": "Structure Definition",
+                "description": "Parent Table",
+                "type": "string",
+                "maxLength": 4,
+                "pattern": "^[0-9]+$"
+              },
+              "parentTable": {
+                "title": "Parent Table",
+                "description": "Parent Table",
+                "type": "string",
+                "maxLength": 30
+              },
+              "dependentTable": {
+                "title": "Dependent table",
+                "description": "Dependent table",
+                "type": "string",
+                "maxLength": 30
+              }
+            },
+            "additionalProperties": false,
+            "required": [
+              "recordNo",
+              "parentTable",
```

Comment:
is this really required? I have seen objects in the sap gui where this field is empty.